src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -2016,8 +2016,24 @@ namespace HVACUnitaryBypassVAV {
 
         // Using/Aliasing
         using namespace DataSizing;
+        using DataAirSystems::PrimaryAirSystem;
         using ReportSizingManager::ReportSizingOutput;
 
+        if (CurSysNum > 0 && CurOASysNum == 0) {
```

Comment:
This is suspicious. Why does CurOASysNum need to be 0? If there were an outdoor air system these flags wouldn't be set.

Reply:
I guess that when the OA Sys flag is true the code is simulating the OA system components and in that case there would be no fan heat.